File: Python/Leetcode/27_remove_element.py
class Solution:
    def removeElement(self, nums, val):
        j =0
        for i in range(len(nums)):
            if nums[i] != val:
                nums[j] = nums[i]
                j += 1
        print(nums)
        return j
        
        
opt = Solution()
print(opt.removeElement([3,2,2,3],3))

File: C++/Leetcode/27_remove_element.cpp
class Solution {
public:
    int removeElement(vector<int>& nums, int val) {
        int count = 0;
        int sz = nums.size();
        
        for (auto i = nums.begin(); i != nums.end(); ++i) {
        if (*i == val) {
            nums.erase(i);
            i--;
            count++;
        }
    }
        
        return sz-count;
    }
};
